### ISSUE TITLE
unable to edit other xml files

### ISSUE THREAD

**Issue by vscaiceanu-1a** [ionic-team/trapeze#78]
Hello,
Using `@trapezedev/configure 3.0.5`, I am trying to update app_name, title_activity_main, package_name and custom_url_scheme in `strings.xml` for Android.

As per the [documentation](https://trapeze.dev/docs/configuration-tool#manifest) I am trying:
```yaml
  android:
    manifest:
      - file: app/src/main/res/values/strings.xml
        target: resources
        merge: |
          <string name="app_name">My awesome app</string>
          <string name="title_activity_main">Awesome activity</string>
          <string name="package_name">com.my.app</string>
          <string name="custom_url_scheme">com.my.app</string>
```
The `strings.xml` file isn't updated.

If I do something like:
```yaml
  android:
    manifest:
      - file: app/src/main/res/values/strings.xml
        target: manifest/application
        inject: |
          <string name="app_name">My awesome app</string>
          <string name="title_activity_main">Awesome activity</string>
          <string name="package_name">com.my.app</string>
          <string name="custom_url_scheme">com.my.app</string>
```

I see the 4 new strings in the `AndroidManifest.xml` file.

I suppose that no matter what file we want to target, `AndroidManifest.xml` will be updated.

**Comment by mlynch** [comment]
This is supported using the `xml` operation. The docs for this were missing and I just added them. Note a similar operation exists for iOS but is missing the delete operations per #79 but I will fix that for the next release https://trapeze.dev/docs/configuration-tool#xml

**Comment by mlynch** [comment]
Fixed in 3.0.6
